datasetinsights/io/tracker/mlflow.py

```diff
@@ -45,28 +45,27 @@ class MLFlowTracker:
     EXP_NAME = "experiment"
     RUN_NAME = "run"
     DEFAULT_RUN_NAME = "run-" + TIMESTAMP_SUFFIX
+    TRACKER = "tracker"
+    MLFLOW_TRACKER = "mlflow"
+    DEFAULT_EXP_NAME = "datasetinsights"
 
-    def __init__(self, mlflow_config):
+    def __init__(self, config):
```

Comment:
[mustfix]

We missed this in the first review. I think it's actually more robust to have host_id, client_id, exp_name and run_name to be the variable passed to the tracker instnace. I would expect the code to be called like this:


```
# factory.py

mlflow_config = config["trackers"]["mlflow"]
mlf_tracker = TrackerFactory.create(**mlflow_config)

# mlflow.py

__init__(self, *, client_id=None, host_id=None, run_name=None, exp_name=None):
...
```

Reply:
This is more pythontic and does not require you to write code such as `mlflow_config.get(MLFlowTracker.HOST_ID)` in order to fetch variables stored in the config dict. It's also easier to maintain as users know what to expect in order to properly initialized an mlflow tracker instance, without looking onto the code itself. Also, how each variable is used, whether it's required or not should be part of the docstring.